src/components/common/DynamicTabs/useTabs.ts

```diff
@@ -289,6 +289,25 @@ export function useTabs(persistanceKey: string) {
         return isTabFound
     }
 
+    const markTabActiveById = (id: string) => {
+        let isTabFound = false
+
+        setTabs((prevTabs) => {
+            const _tabs = prevTabs.map((tab) => {
```

Comment:
The useTabs can be used in multiple places. So this can be picked up later for a refactor with more robust features.
Like the fixed tabs should be available from the initial render itself....